pyincore/analyses/seasidecge/seasidecge.py

```diff
@@ -875,7 +875,7 @@ def _(x):
 
         # create data frame for factor taxes by sector
 
-        a = pd.Series(index=I).fillna(0.0)
+        a = pd.Series(index=I, dtype='float64').fillna(0.0)
         a = SAM.loc[USSOCL, I].append(a, ignore_index=True).append(SAM.loc[GL, I])  # labor, land, capital
```

Comment:
this append needs to be replaced by concat

Reply:
fixed